motr/client.c

```diff
@@ -928,6 +988,31 @@ int32_t m0_rc(const struct m0_op *op)
 }
 M0_EXPORTED(m0_rc);
 
+int m0_client_calculate_pi(struct m0_generic_pi *pi,
```

Comment:
1)m0_generic_pi - IN_OUT variable (IN - > pi_type and 'N-1' context, OUT - N pi_value/checksum)
2)m0_pi_seed - IN (Null always, non-null/real value at the last chunk of motr unit)
3)m0_bufvec *bvec - IN 
4)enum m0_pi_calc_flag flag - IN
5)unsigned char *curr_digest - OUT (Alwauys)
5)unsigned char *checksum_without_seed - OUT (based on flag) (Rename to pi_without_seed)

Reply:
Done.